src/olympia/promoted/migrations/0031_promotedaddon_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.20 on 2025-05-02 18:59
+
+from django.db import migrations
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('versions', '0047_auto_20241031_1750'),
+        ('addons', '0054_update_default_locale_es_to_es-es'),
+        ('promoted', '0030_remove_promotedaddon_addon_and_more'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name='PromotedAddonPromotion',
+            new_name='PromotedAddon',
+        ),
+        migrations.RenameModel(
+            old_name='PromotedAddonVersion',
+            new_name='PromotedApproval',
+        ),
```

Comment:
This is also very risky to do - it will do an sql RENAME.  Instead we manually define the db_table name in the model Meta object, so while the model is renamed in django then underlying database table doesn't change it's name.

(In the rare cases that we care about the database table name there's a longer process of writing to a new table and copying over the old rows, then deleting the old table in a subsequent deploy)